src/components/common/PageHeader/index.tsx

```diff
@@ -23,7 +23,7 @@ const PageHeader = ({
     <Box
       className={css.container}
       sx={{
-        height: `${199 + nameHeight}.5px`,
```

Comment:
This is needed for Retina screens, otherwise the Sidebar header and PageHeader aren't on the same line.

Reply:
It does not align for me using half pixels:

![image](https://user-images.githubusercontent.com/20442784/194016132-6440992a-bf88-446c-8221-35ba2ded745f.png)

Reply:
I see. Could you maybe add a half-pixel padding in CSS via this query? https://css-tricks.com/snippets/css/retina-display-media-query/

Reply:
Can you let me know if the change works?

Reply:
For me, the sidebar header has a height of xyz.4px so its not necessarily .5px